templates/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -267,7 +267,7 @@ public class ApiClient {
         json = new JSON();
 
         // Set default User-Agent.
-        setUserAgent("{{{httpUserAgent}}}{{^httpUserAgent}}OpenAPI-Generator/{{{artifactVersion}}}/java{{/httpUserAgent}}");
```

Comment:
You don't need to modify the template for this.  You can set the HTTP user agent with a command-line flag for the generator.  Example from metal-java (the approach used there is the same as what is used in other Equinix SDKs as well): https://github.com/equinix-labs/metal-java/blob/main/Makefile#L52

Reply:
tanks for the tip :)

Reply:
@ctreatma  adjusted your recommendation